src/main/java/io/ebeaninternal/server/query/CQueryBuilder.java

```diff
@@ -681,6 +681,9 @@ private SqlLimitResponse buildSql(String selectClause, OrmQueryRequest<?> reques
     }
 
     String dbWhere = predicates.getDbWhere();
+    if (stripAlias) {
+      dbWhere = StringHelper.replaceString(dbWhere, "t0.", "");
+    }
```

Comment:
@maTrojak this should be moved a few lines down, otherwise I assume, you will get an NPE here, if `dbWhere == null`
Can you add a test:

```java
Query<Customer> updateQuery = server
      .update(Customer.class)
      .set("status", Customer.Status.ACTIVE)
      .query();
updateQuery.update();
```
Can you do also a fix for ebean 12 and let the PR review from @rBygrave?